src/util.ts

```diff
@@ -1,3 +1,5 @@
+import reactFastCompare from 'react-fast-compare';
```

Comment:
Is it possible to do here?

```js
import isDeepEqual from 'fast-deep-equal/react`;
```

Reply:
@streamich no, i got a bunch of typescript errors, let me dig it up

Reply:
```
(virtualenv3) jack-zhao@Colors-MBP react-use % yarn build:cjs
yarn run v1.22.4
$ tsc
node_modules/fast-deep-equal/react.d.ts:1:1 - error TS1046: Top-level declarations in .d.ts files must start with either a 'declare' or 'export' modifier.

1 const equal: (a: any, b: any) => boolean;
  ~~~~~
import reactFastCompare from 'react-fast-compare';

src/util.ts:1:8 - error TS2395: Individual declarations in merged declaration 'isDeepEqual' must be all exported or all local.

1 import isDeepEqual from 'fast-deep-equal/react';
         ~~~~~~~~~~~

src/util.ts:1:8 - error TS2440: Import declaration conflicts with local declaration of 'isDeepEqual'.

1 import isDeepEqual from 'fast-deep-equal/react';
         ~~~~~~~~~~~

src/util.ts:13:14 - error TS1155: 'const' declarations must be initialized.

13 export const isDeepEqual;
                ~~~~~~~~~~~

src/util.ts:13:14 - error TS2395: Individual declarations in merged declaration 'isDeepEqual' must be all exported or all local.

13 export const isDeepEqual;
                ~~~~~~~~~~~


Found 5 errors.
```

Reply:
from https://github.com/FormidableLabs/react-fast-compare
`This version of react-fast-compare tracks fast-deep-equal@3.1.1.`

Reply:
[This should work](https://github.com/streamich/react-use/pull/1282/files#diff-8cd4968b81985f0efc2053eabc37db6dR1-R13) 🤞  once the latest changes of `fast-deep-equal` are published https://github.com/epoberezkin/fast-deep-equal/issues/69